### PR TITLE
Update pipeline inference snippet to use chat messages

### PR DESCRIPTION
**Pull request by pcuenca**

### CODE REVIEW

**Comment by pcuenca** [comment]
Setting as draft to check out locally and update other posts at the same time

File: gemma.md [hunk 1]
```diff
@@ -119,33 +121,29 @@ pip install -U "transformers==4.38.1" --upgrade
 The following snippet shows how to use `gemma-7b-it` with transformers. It requires about 18 GB of RAM, which includes consumer GPUs such as 3090 or 4090.
 
 ```python
-from transformers import AutoTokenizer
-import transformers
+from transformers import pipeline
```
Comment (osanseviero) [inline gemma.md:124]: I think you can change the pip install above as well, no need to pin to specific version

**Review by osanseviero — APPROVED** [review]
Thanks!

File: gemma.md (hunk 1 above)
Comment (pcuenca) [reply gemma.md:124]: Great call, fixed.

**Review by Rocketknight1 — APPROVED** [review]
This looks great, thanks!

**Pull request merged**